checks.d/vsphere.py

```diff
@@ -633,9 +649,10 @@ def _cache_morlist_raw(self, instance):
             'host_include': instance.get('host_include_only_regex'),
             'vm_include': instance.get('vm_include_only_regex')
         }
+        include_only_marked = instance.get('include_only_marked')
```

Comment:
Should probably default to `False`:
`include_only_marked = instance.get('include_only_marked', False)`

we should also probably also use `is_affirmative()` so the user can be a little more loose in the way they specify if the flag is enabled.